docs/src/components/frame/aside.tsx

```diff
@@ -0,0 +1,33 @@
+import cx from 'classnames'
+import React, { HTMLAttributes, useRef } from 'react'
+
+import Theme, { useTheme } from '@pluralsight/ps-design-system-theme'
+
+import useOnClickOutside from '../../hooks/use-on-click-outside'
+import useOnEscape from '../../hooks/use-on-escape'
+
+import styles from './aside.module.css'
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  onRequestClose: () => void
+  open: boolean
+}
+
+export const Aside: React.FC<Props> = (props) => {
```

Comment:
i'm doing all named exports for components....thoughts?

Reply:
I dig.